tests/DwitTech.AccountService.Core.Tests/Services/ActivationServiceTests.cs

```diff
@@ -10,27 +17,26 @@ namespace DwitTech.AccountService.Core.Tests.Services
 {
     public class ActivationServiceTests
     {
-        [Theory]
-        [InlineData("testcase@gmail.com", "example@gmail.com", "EmailTemplate.html", "Mike", true)]
-        public void SendActivationEmail_Returns_True(string fromMail, string toMail, string templateName, string RecipientName, bool expected)
+       [Theory]
+       [InlineData("2", "testcase@gmail.com", "example@gmail.com", "EmailTemplate.html", "Mike", true)]
+
+        public async Task SendActivationEmail_Returns_True(string userId, string fromMail, string toMail, string templateName, string RecipientName, bool expected)
+        
         {
-            //Arrange
-            var inMemorySettings = new Dictionary<string, string> {
-                {"BaseUrl", "https://example.com"}
-            };
+            var _configuration = new ConfigurationBuilder().AddInMemoryCollection(new Dictionary<string, string>
+            {
+
+
+            }).Build();
```

Comment:
I think you intended to do one thing, but ended up doing something else from your first arrange line to this point.  And the test doesn't seem to cover enough cases.

Reply:
i was a mistake and it have been corrected.